src/main/java/org/wso2/carbon/connector/azure/storage/BlobEraser.java

```diff
@@ -51,8 +51,14 @@ public void connect(MessageContext messageContext) {
         String containerName = messageContext.getProperty(AzureConstants.CONTAINER_NAME).toString();
         String fileName = messageContext.getProperty(AzureConstants.FILE_NAME).toString();
 
+        String protocol = "http";
+        if (messageContext.getProperty(AzureConstants.PROTOCOL) != null) {
```

Comment:
messageContext.getProperty(AzureConstants.PROTOCOL), can we assign this a variable and re-use